### PR TITLE
Homepage redesign

### LINKED ISSUE #615: Link API documentation on homepage
image

https://docs.google.com/document/d/1y-kRDOssYyRwEVTsntqGxoBtvFS4HKMQD-U0Ga9YzJE/preview

### LINKED ISSUE #616: Center homepage layout
For a more single-scroll view, e.g. here's what I have so far
image

### PR DESCRIPTION
**Pull request by MaxGhenis**
* Center-aligns, fixed #616 
* Shows only one flag for users in the US or UK
* Links API, fixes #615 
* Changes some text colors

TODO: Shrink the page width so the flag isn't so huge and so the smaller text isn't so many characters per line.

image

### CODE REVIEW

**Comment by MaxGhenis** [comment]
I made it less blue. The blue version shows up on hover. Still too big, and I think a blue border might make clarify that it's clickable.
image

**Comment by MaxGhenis** [comment]
The latest:
image


image

**Comment by MaxGhenis** [comment]
I think antd buttons would be easiest, as they're already designed for encouraging primary and secondary actions, and they align with the rest of the site. But I couldn't figure out how to make them taller to make room for larger fonts. They don't adjust based on font size.
image

**Comment by MaxGhenis** [comment]
I'm also trying to fix the header alignment. After removing the About and Donate buttons from the top, the social media icons are still not right-aligned:
image
and when I shrink the window such that they're not there, the header bar becomes (unwantedly) taller:
image

**Comment by nikhilwoodruff** [comment]
@MaxGhenis I think the centering issue is due to the `Col` configuration. If you look at the `MainNavigation` component:
```javascript
<Row style={{ margin: 0 }}>
  <Col xl={1} lg={2}>
	  <Title link={props.noTabs && "/"} />
  </Col>
  <Col xl={6} lg={4} className="d-flex align-items-center justify-content-center" style={{ paddingLeft: 25, paddingRight: 25 }}>
	  {middleColumn}
  </Col>
  <Col lg={2} className="d-none d-lg-flex align-items-center justify-content-right">
	  <SocialLinks />
  </Col>
</Row>
```

So when the screen is large (which I think it is in your screenshots), we have 2/6 of the width allocated to the logo, and 4/6 to the social icons. The social icons are centered in their section (which is only the rightmost 2/3s), so that's why it looks uncentered. I've fixed it by removing the XL options (I don't think we're really using them) and setting the lg to 2/8/2.

image

File: policyengine-client/src/landing.jsx [hunk 1]
```diff
@@ -37,60 +34,54 @@ function UsageBox(props) {
...
 
 }
 
 function UsageExplanations(props) {
     return <>
-        <UsageBox title="Citizens" description="Check your eligibility for government benefits and programs. Simulate how a change in requirements could affect your household." />
-        <UsageBox title="Think tanks" description="Simulate tax-benefit reforms on the UK economy and individual households to better understand and recommend policy changes." />
-        <UsageBox title="Parties and campaigns" description="Estimate the economic outcomes of your policy proposals. Produce evidence-based talking points and refute speculative criticism of your platform." />
-        <UsageBox title="Developers" description={<>Allow your users to check their benefits eligibility with our API. <a href="https://github.com/PolicyEngine">Contribute to our work on GitHub.</a></>} />
+        <UsageBox title="The public" description="Estimate your taxes and government benefits, under current law and potential policy changes" />
```
Comment (nikhilwoodruff) [inline policyengine-client/src/landing.jsx:48]: Could we add a centered dot between each of these? Just to separate them, as on mobile at first glance it kind of looks like they might be one multi-line point.

**Comment by MaxGhenis** [comment]
Getting there with a taller button, just not sure how to vertically center the text and enlarge it. Headings could work but then need to change the font color to white.
image

File: policyengine-client/src/landing.jsx (hunk 1 above)
Comment (nikhilwoodruff) [reply policyengine-client/src/landing.jsx:48]: Sorry, my bad - I meant the taglines at the top (estimate your taxes and benefits ... ). Though actually it's pretty marginal

**Comment by nikhilwoodruff** [comment]
Taking a look now, @MaxGhenis just BTW I'd avoid using `<br />`s, and instead use `style={{marginTop: x}}` on the components themselves.

**Pull request merged**